app/controllers/katello/api/v2/repository_sets_controller.rb

```diff
@@ -41,7 +41,6 @@ def show
     def available_repositories
       scan_cdn = sync_task(::Actions::Katello::RepositorySet::ScanCdn, @product, @product_content.content.cp_content_id)
       repos = scan_cdn.output[:results]
-
```

Comment:
Let's put the empty line back. I think judicious use of newlines makes code more readable within our rubocop guidelines of course!